keyboards/lily58/keymaps/oldfieldtc/config.h

```diff
@@ -0,0 +1,50 @@
+/*
+This is the c configuration file for the keymap
+
+Copyright 2012 Jun Wako <wakojun@gmail.com>
```

Comment:
Please update this to include an updated copyright date, and reference yourself as author (since it's your keymap).